test/qtismtest/data/AssessmentTestTest.php

```diff
@@ -114,4 +115,168 @@ public function testIsExclusivelyLinear()
         $testPart->setNavigationMode(NavigationMode::LINEAR);
         $this->assertTrue($test->isExclusivelyLinear());
     }
+    
+    public function testGetPossiblePaths()
+    {
+        $doc = new XmlDocument();
+        $doc->load(self::samplesDir() . 'custom/tests/branchingpath.xml');
+        $test = $doc->getDocumentComponent();
+
+        $itemq1 = $doc->getDocumentComponent()->getComponentByIdentifier('Q01');
+        $itemq2 = $doc->getDocumentComponent()->getComponentByIdentifier('Q02');
+        $itemq3 = $doc->getDocumentComponent()->getComponentByIdentifier('Q03');
+        $itemq4 = $doc->getDocumentComponent()->getComponentByIdentifier('Q04');
+        $itemq5 = $doc->getDocumentComponent()->getComponentByIdentifier('Q05');
+        $itemq6 = $doc->getDocumentComponent()->getComponentByIdentifier('Q06');
+
+        $possible_paths = array();
```

Comment:
I know it's a detail but I would like to see $possible_paths = []; for consistency purposes 

(even if the old code in the whole QTI-SDK does not respect that everywhere, let's build on clean stuff!)

_**Note: there are other places where the same issue ocurs.**_